mount/losetup_linux.go

```diff
@@ -32,8 +35,26 @@ const (
 	loopDevFormat   = "/dev/loop%d"
 
 	ebusyString = "device or resource busy"
+
+	LOOP_CONFIGURE = 0x4c0a //nolint: revive
```

Comment:
This const doesn't need to be exported and the convention would be to use [mixed caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps).

```suggestion
	loopConfigureIoctl = 0x4c0a //nolint: revive
```

Reply:
Fixed.